src/App.js

```diff
@@ -1,7 +1,14 @@
 import React from "react";
+import GlobalStyles from "./GlobalStyles.js";
```

Comment:
You dont need to define the .js extension here.

Reply:
thanks ;*